crates/ironclaw_webui_v2_static/static/js/i18n/ko.js

```
@@ -657,36 +657,74 @@ registerPack("ko", {
   "automations.filterLabel": "자동화 상태 필터",
   "automations.filter.all": "All",
   "automations.filter.active": "활성",
-  "automations.filter.running": "Running",
-  "automations.filter.failures": "Failures",
+  "automations.filter.running": "실행 중",
```

Biome (2.5.0)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)